### PR TITLE
[DOCS] Update how-to-create-and-edit-expectations-in-bulk.md

### PR DESCRIPTION
**Pull request by petermoyer**
Update stub page

Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- GDOC-193 - Add missing page

### Definition of Done
Please delete options that are not relevant.

- [ x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [x ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ x] I have made corresponding changes to the documentation

Thank you for submitting!

### CODE REVIEW

File: docs/guides/expectations/how-to-create-and-edit-expectations-in-bulk.md [hunk 1]
```diff
@@ -2,4 +2,96 @@
...
+````console
+jsonschema_file = "YOUR_JSON_SCHEMA_FILE.json"
+suite_name = "YOUR_SUITE_NAME"
+````
+
+#### 2. Load a DataContext
+
+````console
+context = ge.data_context.DataContext()
+````
```
Comment (cdkini) [inline docs/guides/expectations/how-to-create-and-edit-expectations-in-bulk.md]: Please change these both to `python`

File: docs/guides/expectations/how-to-create-and-edit-expectations-in-bulk.md [hunk 2]
```diff
@@ -2,4 +2,96 @@
...
+
+#### 7. Optionally, generate Data Docs and review the results there.
+
+Data Docs provides a concise and useful way to review the Expectation Suite that has been created.
+
+````console
+   context.build_data_docs()
+
+You can also review and update the Expectations created by the profiler to get to the Expectation Suite you want using ``great_expectations suite edit``.
+````
```
Comment (cdkini) [inline docs/guides/expectations/how-to-create-and-edit-expectations-in-bulk.md]: Please left-align `context.build_data_docs()` so it matches the rest of the code blocks. Additionally, the following sentence shouldn't be wrapped in the code block right?

File: docs/guides/expectations/how-to-create-and-edit-expectations-in-bulk.md [hunk 3]
```diff
@@ -2,4 +2,96 @@
...
+
+### Additional notes
+
+:::note Important
+
+Note that JsonSchemaProfiler generates Expectation Suites using column map expectations, which assumes a tabular data structure, because Great Expectations does not currently support nested data structures.
+:::
```
Comment (cdkini) [inline docs/guides/expectations/how-to-create-and-edit-expectations-in-bulk.md]: Screen Shot 2021-07-19 at 1 24 14 PM

Tags like `Important`, `Tip`, and `Warning` should not have `note` in the titles. Please see the above screenshot for what `Important` looks like without note usage. `Prerequisites` renders the same so it's not a concern but all other tags are impacted.

**Review by cdkini — CHANGES_REQUESTED** [review]
Minor changes requested. Should be very brief! Please let me know when you want me to take an additional look 🚀

File: docs/guides/expectations/how-to-create-and-edit-expectations-in-bulk.md [hunk 4]
```diff
@@ -2,4 +2,96 @@
 title: ✳ How to create and edit Expectations in bulk
 ---
 
-This article is a stub.
+The `JsonSchemaProfiler` helps you quickly create [Expectation Suites](/docs/reference/expectation-suite-operations) from jsonschema files.
+
+:::note Prerequisites 
+This how-to guide assumes you have already:
+
```
Comment (cdkini) [inline docs/guides/expectations/how-to-create-and-edit-expectations-in-bulk.md]: After giving this some more thought, can we convert this to the `<Prerequisites>` React component? They both look the same when rendered but Taylor added error handling and a few other bits of functionality that I'd love to have in here just for consistency.

**Review by petermoyer — COMMENTED** [review]
Check again

**Review by cdkini — APPROVED** [review]
LGTM

**Pull request merged**